app/views/posts/index.html.erb

```diff
@@ -1,3 +1,30 @@
+<p style="color: green"><%= notice %></p>
 
-<h1>Posts</h1>
+<div class = "container my-5 userBG">
+    <div class = "row">
+      <div class="col-12 row">
+        <img src= "<%= @user.photo %>" class="col-6 py-2 userImage">
+        <div class = "col-6 row py-5">
+          <h3 class = "col-12"><%= @user.name %></h3>
+          <h4 class = "col-12">Number of posts:  <%= @user.posts_counter %></h4>
+        </div>
+    </div>
+  </div>
+</div>
 
+<div id="posts" class= "container userBG">
+  <% @posts.each do |post| %>
+    <%= render post %>
+    <p class="ebdShowPost">
+      <%= link_to "Show this post", user_post_path(@user, post) %>
+      <h4 class="text-light py-3">Comments</h4>
+      <% post.recent_comments.each do |comment|%>
+        <div>
+            <span><%= post.author.name %>:</span>
+            <span><%= comment.text %></span>
+          </div>
+      <% end %>
+    </p>
+    <hr>
+  <% end %>
```

Comment:
- [ ] it would be better to match the wireframe given  in order to match the requirements 
- wireframe 
<img width="515" alt="Screenshot 2022-12-17 at 13 29 19" src="https://user-images.githubusercontent.com/61790587/208241883-84f8b434-5cf8-4d13-ae14-93fffe770575.png">
- Yours 
<img width="289" alt="Screenshot 2022-12-17 at 13 24 22" src="https://user-images.githubusercontent.com/61790587/208241922-98723be9-8f10-4d6c-8e4b-fb3241c3c048.png">

Reply:
hi @youmari 
It is working in local port

Reply:
@youmari it's working for me too but the style doesn't match the wireframe